### ISSUE TITLE
The prefect generates a large amount of zoombie processes

### ISSUE THREAD

**Issue by semcha** [PrefectHQ/prefect#5481]
## Description

I have two independent different servers with prefect and both of them generate a thousands of zoombie processes.
When I stop server by `prefect server stop`,  they disappear.

## Reproduction

I start the server with the following commands:
```
nohup prefect server start --expose --use-volume &
nohup prefect agent local start --label prod --no-hostname-label --import-path /home/my_user/my_project/ &
```

## Environment

Ubuntu 20.04
```{
  "config_overrides": {
    "context": {
      "secrets": false
    }
  },
  "env_vars": [],
  "system_information": {
    "platform": "Linux-5.4.0-100-generic-x86_64-with-glibc2.29",
    "prefect_backend": "server",
    "prefect_version": "1.0.0",
    "python_version": "3.8.10"
  }
}

**Comment by semcha** [comment]
I watch the processes with the command `ps aux | grep 'Z'`.
I started the server an hour and a half ago and already have 500+ zoombie processes.
Example output:
```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root        2607  0.0  0.0      0     0 ?        Z    19:27   0:00 [sh] <defunct>
root        2694  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        2768  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        2846  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        2926  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        2982  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        3075  0.0  0.0      0     0 ?        Z    19:28   0:00 [sh] <defunct>
root        3130  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3208  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3280  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3353  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3418  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3489  0.0  0.0      0     0 ?        Z    19:29   0:00 [sh] <defunct>
root        3564  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        3620  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        3717  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        3776  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        3857  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        3930  0.0  0.0      0     0 ?        Z    19:30   0:00 [sh] <defunct>
root        4015  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4069  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4161  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4225  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4300  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4374  0.0  0.0      0     0 ?        Z    19:31   0:00 [sh] <defunct>
root        4448  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4500  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4593  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4645  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4721  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4794  0.0  0.0      0     0 ?        Z    19:32   0:00 [sh] <defunct>
root        4870  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        4943  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        5017  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        5070  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        5165  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        5217  0.0  0.0      0     0 ?        Z    19:33   0:00 [sh] <defunct>
root        5290  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5381  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5479  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5533  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5630  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5713  0.0  0.0      0     0 ?        Z    19:34   0:00 [sh] <defunct>
root        5786  0.0  0.0      0     0 ?        Z    19:35   0:00 [sh] <defunct>
```

**Comment by semcha** [comment]
Tried different versions. In version `0.15.10` and `0.15.11` everything is fine and there is no such problem. Zoombie processes appear since version `0.15.12`.

**Comment by madkinsz** [comment]
This is closed by #5479

**Comment by semcha** [comment]
@madkinsz Thank you so much for your response, I'm waiting for the release 🙂
